Gemfile

```diff
@@ -75,13 +79,13 @@ gem 'audio_waveform-ruby', '~> 1.0.7', require: 'audio_waveform'
 gem 'browse-everything', git: "https://github.com/avalonmediasystem/browse-everything.git", branch: 'v1.2-avalon'
 gem 'fastimage'
 gem 'media_element_add_to_playlist', git: 'https://github.com/avalonmediasystem/media-element-add-to-playlist.git', tag: 'avalon-r6.5'
-gem 'mediainfo', git: "https://github.com/avalonmediasystem/mediainfo.git", branch: 'avalon_fixes'
+gem 'mediainfo', git: "https://github.com/avalonmediasystem/mediainfo.git", branch: 'rails6.1'
```

Comment:
We should explore upgrading and moving off of our fork, but for now this doesn't change what we've been doing.